packages/react-ui-components/src/MultiSelectBox_ListPreviewSortable_DraggableListPreviewElement/multiSelectBox_ListPreviewSortable_DraggableListPreviewElement.js

```diff
@@ -129,7 +137,11 @@ export default class MultiSelectBox_ListPreviewSortable_DraggableListPreviewElem
                             hoverStyle={'clean'}
                             />
                     )}
-                    <div className={theme.selectedOptions__innerPreview}>
+                    <div
+                        className={theme.selectedOptions__innerPreview}
+                        onClick={option.uri ? () => handleClick(option.uri) : undefined }
```

Comment:
How about forwarding the whole `option` here and let the outside handle do whatever it wants with it?
Then we don't the to know about an `uri`.

Reply:
I would directly call `handleClick` and as I said before, pass the whole option to the callback without checking for an uri. Let the callback do whatever (f.e. showing a modal if we need one later on.